packages/kit/types.internal.d.ts

```diff
@@ -88,16 +88,21 @@ export type Headers = Record<string, string>;
 
 export type LoadInput = {
 	page: Page;
-	fetch: (info: RequestInfo, init?: RequestInit) => Promise<NodeFetchResponse>;
```

Comment:
It seems `NodeFetchResponse` was change to just `Response` making the import unused, is this intentional?

Reply:
It is, and it was removed in a recent cleanup PR